### DISCUSSION TITLE
Use Web API instead

### DISCUSSION THREAD

**Discussion by mecuricat** [robertodoering/harpy#891] [Ideas]
Instead of the official Twitter API, [Fritter](https://github.com/jonjomckay/fritter) uses the API that the web interface uses.

What are your thoughts on using that to revive the project?

**Comment by robertodoering** [comment]
I've answered a similar suggestion here: #888

> There are a few reasons why this isn't an option:
>
> - Relying on undocumented api's or scraping content is unreliable and prone to stop working at any point in time. I don't have time to maintain it.
> - This particular library is missing many endpoints we'd need for a proper client.
> - Third party clients are still officially banned by Twitter. They can and probably would take actions to have them removed from any official store.
